cloud-etl/statements.sql

```diff
@@ -1,3 +1,3 @@
 CREATE STREAM eventlogs (eventSourceIP varchar, eventAction varchar, Result varchar, eventDuration bigint) WITH (kafka_topic='eventlogs', value_format='JSON');
 CREATE TABLE count_per_source WITH (KAFKA_TOPIC='COUNT_PER_SOURCE', PARTITIONS=6) AS SELECT eventSourceIP, count(*) as COUNT FROM eventlogs GROUP BY eventSourceIP EMIT CHANGES;
-CREATE TABLE sum_per_source WITH (KAFKA_TOPIC='SUM_PER_SOURCE', PARTITIONS=6, VALUE_FORMAT='AVRO') AS SELECT eventSourceIP, sum(eventDuration) as SUM FROM eventlogs WHERE Result='Pass' GROUP BY eventSourceIP EMIT CHANGES;
+CREATE TABLE sum_per_source WITH (KAFKA_TOPIC='SUM_PER_SOURCE', PARTITIONS=6, VALUE_FORMAT='AVRO') AS SELECT eventSourceIP ROWKEY, as_value(eventSourceIP) EVENTSOURCEIP, SUM(EVENTDURATION) SUM FROM eventlogs WHERE (RESULT = 'Pass') GROUP BY EVENTSOURCEIP EMIT CHANGES;
```

Comment:
For my own education, why is `AS` not required?  i.e., how is it that `sum(eventDuration) as SUM` changed to `SUM(EVENTDURATION) SUM`

Reply:
I believe the `AS` can be implied in most SQL dialects, but we should be consistent ( I didn't notice they were missing here, thank you).  I'll add `AS` across the board and make sure the casing consistent as well.

Reply:
Updated statements.sql, please check me.  thank you